main.tex

```diff
@@ -39,7 +39,7 @@
 
 \section{Protocol Overview}
 
-\textit{Note that the following is an incomplete overview, in its current form it only attempts to give some intution for the cryptographic details described below.}
+\textit{Note that the following is an incomplete overview, in its current form it only attempts to give some intution for the cryptographic details described below. It is limited in scope to only the coomunication between user and server, and disregards the final CoinJoin transaction structure.}
```

Comment:
NACK. It's limited in many ways as for now.

Reply:
I know - but when I asked for feedback of some friends _all of them_ have brought up that it can be deanonymized in the tx construction. so I think it would be good to mention this part of the limited scope explicitly.